modCrypto/src/org/aion/crypto/AddressSpecs.java

```diff
@@ -25,4 +26,31 @@ private AddressSpecs() {}
         buf.put(HashUtil.h256(publicKey), 1, 31);
         return buf.array();
     }
+
+    public static String checksummedAddress(String address) {
```

Comment:
Can we change the method signature to return Optional<String>?  And in places where we currently return null, return `Optional.empty()` instead?